modules/abstract-cosmos/src/cosmosCoin.ts

```diff
@@ -382,8 +382,8 @@ export class CosmosCoin extends BaseCoin {
     const MESSAGE = Buffer.from(txHex, 'hex');
 
     const [signA, signB] = [
-      MPC.sign(MESSAGE, signCombineOne.oShare, signCombineTwo.dShare, createHash('sha256')),
-      MPC.sign(MESSAGE, signCombineTwo.oShare, signCombineOne.dShare, createHash('sha256')),
+      MPC.sign(MESSAGE, signCombineOne.oShare, signCombineTwo.dShare, this.getHashFunction()),
```

Comment:
I think it wouldn't, since this uses `keccak256`. And it seems in OVC we default to `keccak256` unless explicitly [added here](https://github.com/BitGo/offline-vault-console/blob/d22166c5384dfd5fb4aeac7c77a54430615a24b9/frontend/src/pkg/bitgo/utils.ts#L182C1-L182C1)